### ISSUE TITLE
Default Closed Captions Button Visibility Toggle to On

### ISSUE THREAD

**Issue by viscountexx** [crackededed/Xtra#335]
Closed captions being hidden away in a menu is a major accessibility issue. The menus are complex and I expect the defaults to be that "extraneous" or "annoying" buttons like follow (which i can access outside the player anyway) are turned off but standardized accessibility features that are an option on literally every single TV/media device should be shown by default. 

(Edit to clarify: I would like the button to be visible by default, not captions to be on by default.)

I didn't even know this option existed until checking GitHub.

**Comment by thinkpad4** [comment]
Make the option easier to find but do not make it default on

**Comment by mindmeshlink** [comment]
> Make the option easier to find but do not make it default on

Why?

**Comment by thinkpad4** [comment]
Cause that's one extra thing I have to do then

**Comment by thinkpad4** [comment]
@mindmeshlink That makes SO much more sense.  Totally agree that the option should be visible by default.

**Comment by crackededed** [comment]
currently the player can't detect if the video has captions or not so the button is always forced to be visible with this setting on. so most of the time you have a button that does nothing. maybe it would be better to have this setting visible in the player menu by default. also chunkless preparation is disabled to force the captions to work. this increases start up time for the player. i think it should only be a few seconds at most but maybe it causes problems for older devices. so i don't really want to disable that by default. maybe the playlist could be modified instead to enable the captions and then that would not be a problem.

**Comment by crackededed** [comment]
captions work with chunkless preparation now so i've made the subtitles option in the player menu visible by default
